next.config.mjs

```diff
@@ -2,6 +2,14 @@
 
 const nextConfig = {
   transpilePackages: ["@penumbra-zone/protobuf"],
+  experimental: {
+    swcPlugins: [
+      [
+        "@preact-signals/safe-react/swc",
```

Comment:
this plugin helps all react components understand if the signals are used inside it – and subscribe to signal updates